### ISSUE TITLE
Bootstrap doesn't work

### ISSUE THREAD

**Issue by blinq-kim** [blinqas/station#67]
## Bugs
- `github_repository.name`  points to `var.vcs_repo_identifier`
- An input variable with the name "tfe_organization_namer" has not been declared.
- An input variable with the name "vcs_repo_PAT" has not been declared.
- An input variable with the name "tfc_team_token" has not been declared.
- `var.vcs_repo_identifier` is not set in `bootstrap.sh`

## Enhancement
- `azuread_application.display_name` is `app-station-tfe-tests` by default. Change to `app-station-bootstrap`.
- Show selected azure subscription as well as tenant id on `Azure Account Information`.

**Comment by Blinq-SanderBlom** [comment]
- **github_repository.name points to var.vcs_repo_identifier:** This has now been changed to point to vcs_repo_name and the vcs_repo_identifier has been removed because it was previously used before the repository was created during the bootstrap 
- **An input variable with the name "tfe_organization_namer" has not been declared.:** This has not been declared in the variables.tf but I cant find any code reference to this anywhere in the codebase. Is this needed ? 
- **An input variable with the name "vcs_repo_PAT" has not been declared:** Added in [commit](2f2df59e0ab72bd18b51f796635d619494ac3209).
- **An input variable with the name "tfc_team_token" has not been declared:** Renamed this to tfc_token to match value in bootstrap script. This can also be an org token so the name was not correct. Fixed in [commit](2f2df59e0ab72bd18b51f796635d619494ac3209)

**Comment by ghost** [comment]
## Bugs
- **github_repository.name points to var.vcs_repo_identifier:** This has now been changed to point to vcs_repo_name and the vcs_repo_identifier has been removed because it was previously used before the repository was created using terraform
- **An input variable with the name "tfe_organization_namer" has not been declared.:** This has not been declared in the variables.tf but I cant find any code reference to this anywhere in the codebase. Is this needed ? 
- **An input variable with the name "vcs_repo_PAT" has not been declared:** Added in 2f2df59e0ab72bd18b51f796635d619494ac3209.
- **An input variable with the name "tfc_team_token" has not been declared:** Renamed this to tfc_token to match value in bootstrap script. This can also be an org token so the name was not correct. Fixed in 2f2df59e0ab72bd18b51f796635d619494ac3209
- **~~var.vcs_repo_identifier is not set in bootstrap.sh~~:**

## Enhancement 
- **azuread_application.display_name is app-station-tfe-tests by default. Change to app-station-bootstrap:** This was changed in 5fa090e7fb4a77b744931be1522b9d4d835ca376
- **Show selected azure subscription as well as tenant id on Azure Account Information:** This was added in 1e488a09c927a908eae2dfbc63d994c19a8ed8cd

**Comment by ghost** [comment]
I also fixed the depreciation warning for the `default_brach` attribute in 7c753a9edb5ecee1e6a68e902eb381d21c59209d
```
╷
│ Warning: Deprecated attribute
│ 
│   on tfe_workspace.tf line 18, in resource "tfe_workspace" "deployments":
│   18:     branch                     = github_repository.deployments.default_branch
│ 
│ The attribute "default_branch" is deprecated. Refer to the provider documentation for details.
```

**Comment by blinq-kim** [comment]
Resolved in #68
